astropy/table/pprint.py

```diff
@@ -185,7 +185,7 @@ def _get_pprint_size(max_lines=None, max_width=None):
 
 
     def _pformat_col(self, col, max_lines=None, show_name=True, show_unit=None,
-                     show_dtype=False, show_length=None, html=False):
+                     show_dtype=False, show_length=None, html=False, align='right'):
```

Comment:
Never mind the previous comment about the `align` keyword (which I have now deleted).  I think there *should* be an `align` keyword here, and it should override the existing align if that is found in the `col.format`.  The `align` keyword should also be added to the public `Column.pformat` method.